src/main/java/org/terasology/signalling/componentSystem/SignalSystem.java

```diff
@@ -273,6 +295,11 @@ private void outputSignalToSimpleConsumer(EntityRef entity, boolean result) {
         }
     }
 
+    /**
+     * Detects a XOR signal based on the sides receiving a signal
```

Comment:
I would just add: Used for the XOR signal block.

Reply:
Oh boy, I have no clue how I missed this. Definitely agree with MPoll. Don't be lazy :P

Reply:
I wasn’t being lazy. I thought he said that was excessive.

Reply:
lazy? @Steampunkery just removed the full comment from the other two. not sure why?